### PR TITLE
Wagtail 5.1 upgrade

### PR DESCRIPTION
**Pull request by katdom13**
Support ticket: https://torchbox.monday.com/boards/1124794299/pulses/1124795993

[Wagtail 5.1 release notes](https://docs.wagtail.org/en/stable/releases/5.1.html)

Upstream PR: https://github.com/ibrahimawadhamid/wagtail_blocks/pull/13

### Update wagtailfontawesomesvg icons following module update from ~0.0.4 to ~1.0.1
The newest version of `wagtailfontawesomesvg` (1.0.1) has had some icon design changes and icon renames since 0.0.4

https://github.com/wagtail-nest/wagtail-font-awesome-svg/tree/0.0.4/wagtailfontawesomesvg
https://github.com/wagtail-nest/wagtail-font-awesome-svg/tree/1.0.1/wagtailfontawesomesvg

### CODE REVIEW

File: wagtail_blocks/__init__.py [hunk 1]
```diff
@@ -1 +1 @@
-__version__ = "0.8.1"
+__version__ = "1.0.1"
```
Comment (nickmoreton) [inline wagtail_blocks/__init__.py]: @katdom13 We should probably keep the maintainers version here as we won't be releasing it officially. I don't think it matters even if we end up using our own fork in a site.

**Review by nickmoreton — APPROVED** [review]
@katdom13 I approve this but please see my comment about the version. Thanks

File: wagtail_blocks/__init__.py (hunk 1 above)
Comment (nickmoreton) [reply wagtail_blocks/__init__.py]: I also pulled in the latest to the master branch, I don't think it affects your work though.

Comment (katdom13) [reply wagtail_blocks/__init__.py]: Hi @nickmoreton ,
That's actually the maintainer's change. I changed the target branch to main, and this shouldn't show up anymore.
Thanks for checking!

**Pull request merged**